operators/constellation-node-operator/config/manager/kustomization.yaml

```diff
@@ -5,6 +5,12 @@ generatorOptions:
   disableNameSuffixHash: true
 
 configMapGenerator:
-- name: manager-config
-  files:
+- files:
   - controller_manager_config.yaml
+  name: manager-config
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: ghcr.io/edgelesssys/constellation/node-operator
+  newTag: v0.0.1
```

Comment:
This change can be reverted. I think this happens automatically on some make targets but does not make a whole lot of sense (we don't deploy a tag v0.0.1).